docs/core/rid-catalog.md

```diff
@@ -66,7 +66,7 @@ All RIDs eventually map back to the root `any` RID.
 
 There are some considerations about RIDs that you have to keep in mind when working with them:
 
-- RIDs are **opaque strings** and should be treated as black boxes.
+- RIDs are **opaque strings** and should be treated as opaque boxes.
```

Comment:
@adegeo Do you know what this means by "opaque string" and what it means to "treat a RID as a black box"?

Reply:
No idea. @richlander @KathleenDollard

Reply:
Sent message to PM

Reply:
@tdykstra 
Response was that we can choose to hold on the PR for more feedback from PMs or accept the PR and open an issue to define opaque strings. I vote to open a new issue and accept the PR.

Reply:
Given that the sentence is meaningless to everyone who has read it so far, I would prefer to modify this PR to remove the line instead of changing the wording.

Reply:
-- This is an identification format that is not intended to be parsed to retrieve the component parts.

Reply:
Do you mean that a user shouldn't look to a RID to determine platform/os because there isn't a guarntee that the rid is one that identifies that component? Such as using `win` or `osx.10.10-x64` for a rid. The `osx` one has all the information needed to be parsed, however, the generic `win` doesn't.
??